src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
this font is already installed on the body, so it will automatically be applied to all elements, so there is no need to prescribe it to individual elements